src/test/groovy/com/jaredsburrows/spoon/BaseSpec.groovy

```diff
@@ -9,7 +9,7 @@ import spock.lang.Specification
 class BaseSpec extends Specification {
   @Rule TemporaryFolder testProjectDir = new TemporaryFolder()
   def COMPILE_SDK_VERSION = 27
-  def BUILD_TOOLS_VERSION = "27.0.1"
+  def BUILD_TOOLS_VERSION = "27.0.3"
```

Comment:
If you change this here, you need/should change it in the `travis.yml`.

Reply:
I changed here because build tools were already updated to 27.0.3 in `build.gradle` and `travis.yml`.